clients/src/test/java/org/apache/kafka/common/network/SslTransportLayerTest.java

```diff
@@ -129,14 +117,14 @@ public void testInvalidEndpointIdentification() throws Exception {
     public void testEndpointIdentificationDisabled() throws Exception {
         String node = "0";
         String serverHost = InetAddress.getLocalHost().getHostAddress();
-        server = new SslEchoServer(sslServerConfigs, serverHost);
+        server = new NioEchoServer(SecurityProtocol.SSL, sslServerConfigs, serverHost);
         server.start();
```

Comment:
This one overrides the host address to test that endpoint identification can be disabled.